### PR TITLE
Add support for high quality 192k mp3 for Pandora One

### PR DESCRIPTION
**Pull request by ericmason**
In the response for station.getPlaylist the real high quality file is in a key called audioUrlMap like so:

`			"audioUrlMap": {
				"highQuality": {
					"bitrate": "192",
					"encoding": "mp3",
					"audioUrl": "...",
					"protocol": "http"
				},
				"mediumQuality": {
					"bitrate": "64",
					"encoding": "aacplus",
					"audioUrl": "...",
					"protocol": "http"
				}
			},`

The additionalAudioUrl list that the app currently uses only has files up to 128k.

### CODE REVIEW

File: Sources/Pandora/Pandora.m [hunk 1]
```diff
@@ -547,7 +559,7 @@ - (BOOL) fetchStationInfo:(Station *)station {
     created.year = 2000 + [dateCreated[@"year"] integerValue];
     created.month = 1 + [dateCreated[@"month"] integerValue];
     created.day = [dateCreated[@"date"] integerValue];
-    NSCalendar *gregorian = [[NSCalendar alloc] initWithCalendarIdentifier:NSGregorianCalendar];
+    NSCalendar *gregorian = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
```
Comment (reedloden) [inline Sources/Pandora/Pandora.m]: This seems unrelated. Separate PR for it?

Comment (ericmason) [inline Sources/Pandora/Pandora.m]: Yes, it's unrelated, I missed it when submitting. My bag.

Comment (ericmason) [inline Sources/Pandora/Pandora.m]: Bad rather

**Comment by nriley** [comment]
FWIW this actually downgraded quality for non-Pandora One users, which I fixed in 59f095cee88292481f7716000b1dd28e7797e1b3. This is tricky and weird but a good reason to test this stuff as both a Pandora One user and…not.

**Comment by ericmason** [comment]
Wow, that sucks. I will admit, I totally didn't test it as a non-Pandora One user. I was way too excited about being able to use Pandora again to make sure it didn't mess things up elsewhere. Sorry about that, and thanks for all your work on Hermes.
